src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1552,7 +1552,7 @@ export class SavedObjectsRepository {
 
     let bulkGetRequestIndexCounter = 0;
     const expectedBulkGetResults: Either[] = objects.map((object) => {
-      const { type, id } = object;
+      const { type, id, workspaces } = object;
```

Comment:
There is already a workspaces field in options, what are the differences between the object one?

Reply:
I will use addToWorkspace api instead of bulkUpdate api.